### ISSUE TITLE
Panic when executing reply action without contact

### ISSUE THREAD

**Issue by ericnewcomer** [nyaruka/goflow#152]
Should instead return a friendly error.
```
runtime/debug.Stack(0xc420500e68, 0x1720760, 0x217f370)
	/usr/local/go/src/runtime/debug/stack.go:24 +0x79
github.com/nyaruka/goflow/vendor/github.com/pressly/lg.RequestLogger.func1.1.1(0xc42045ca60, 0x21434c0, 0xc420142f80, 0xed2279d75, 0x22713320, 0x218f1e0)
	/Users/eric/code/go/src/github.com/nyaruka/goflow/vendor/github.com/pressly/lg/middleware.go:30 +0x14e
panic(0x1720760, 0x217f370)
	/usr/local/go/src/runtime/panic.go:489 +0x2cf
github.com/nyaruka/goflow/flows/actions.(*ReplyAction).Execute(0xc4204e4300, 0x214dda0, 0xc4205fc000, 0x2143fa0, 0xc420112850, 0x213bde0, 0xc420353440, 0x0, 0x0)
	/Users/eric/code/go/src/github.com/nyaruka/goflow/flows/actions/reply.go:46 +0x176
github.com/nyaruka/goflow/flows/engine.(*session).visitNode(0xc42008ca00, 0x214dda0, 0xc4205fc000, 0x21404c0, 0xc4204e42a0, 0xc420379640, 0x1, 0x1, 0xc4205fc000, 0xc4205012a8, ...)
```

**Comment by rowanseymour** [comment]
We've got a few actions which don't make so much sense without a contact - and currently some of them noop. But error does seem more appropriate.

**Comment by ericnewcomer** [comment]
For sure, just making sure we log cases where the caller can blow things up with unexpected input so we can return error messages instead.

**Comment by rowanseymour** [comment]
Fixed in #175
